submarineswap.go

```diff
@@ -11,10 +11,10 @@ import (
 	"swapper/mempoolspace"
 
 	"github.com/btcsuite/btcd/btcec"
+	"github.com/btcsuite/btcd/btcutil"
 	"github.com/btcsuite/btcd/chaincfg"
 	"github.com/btcsuite/btcd/txscript"
 	"github.com/btcsuite/btcd/wire"
-	"github.com/btcsuite/btcutil"
```

Comment:
It seems that all the functions in this file should not be exported as they are used internally only.